.github/workflows/CI.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
         julia-arch: [x64, x86]
-        julia-version: ['1','1.6-nightly', 'nightly']
+        julia-version: ['1','nightly']
```

Comment:
Perhaps you want to also test the oldest supported version (1.3)?  It's occasionally too easy to accidentally break something for an older version just because CI didn't catch it

Reply:
true. Thanks